developer-guides/rest-api/README.md

```diff
@@ -65,35 +65,37 @@ When calling a production Rocket.Chat server, ensure it is running via HTTPS and
 | `/api/v1/channels.list.joined`     | Gets only the channels the calling user has joined.         | [Link](channels/list-joined/)     |
 | `/api/v1/channels.open`            | Adds the channel back to the user's list of channels.       | [Link](channels/open/)            |
 | `/api/v1/channels.rename`          | Changes a channel's name.                                   | [Link](channels/rename/)          |
+| `/api/v1/channels.setCustomFields` | Sets a channel's custom fields.                             | [Link](channels/setcustomfields/) |
 | `/api/v1/channels.setDescription`  | Sets a channel's description.                               | [Link](channels/setdescription/)  |
 | `/api/v1/channels.setJoinCode`     | Sets the channel's code required to join it.                | [Link](channels/setjoincode/)     |
-| `/api/v1/channels.setPurpose`      | Sets a channel's description.                               | [Link](channels/setpurpose/)      |
```

Comment:
Please let this be as it was before, the api is called set purpose due compatibility reasons, it does really change the description of the channel (as shown in the ui)

Reply:
I see. Restored with an obsolescence notification.